packages/api/lib/routes.js

```diff
@@ -74,6 +74,7 @@ const routes = {
   'noauth': [
     { path: '/', method: 'get', handler: printRoutes, reference: 'api-Misc-printRoutes' },
     { path: '/stats', method: 'get', handler: statisticsController.getStatistics, reference: 'api-Misc-getStatistics' },
+    { path: '/getAllTags', method: 'get', handler: boxesController.getAllTags, reference: 'api-Boxes-getAllTags' },
```

Comment:
Just call the route `/tags`.
The method already indicates that it is a `GET` request.
`All` is specifying a parameter. If it is needed pass it as a parameter like `?all=true`